src/app/tracing/graph/cy-graph/interactive-cy-graph.ts

```diff
@@ -214,7 +214,7 @@ export class InteractiveCyGraph extends CyGraph {
         return Math.min(Math.max(zoom, this.minZoom), this.maxZoom);
     }
 
-    protected zoomTo(zoom: number, position?: Position): void {
+    protected zoomWithCusorAt(zoom: number, position?: Position): void {
```

Comment:
I guess there is a typo. `Cursor`--> `Cursor`. What does `zoomWithCusorAt` mean?